doc/man1/ciphers.pod

```diff
@@ -156,19 +156,20 @@ The cipher string B<@SECLEVEL=n> can be used at any point to set the security
 level to B<n>, which should be a number between zero and five, inclusive.
 See L<SSL_CTX_set_security_level> for a description of what each level means.
 
+The cipher list can be prefixed with the B<DEFAULT> keyword, which enables
+the default cipher list as defined below.  Unlike the cipher string,
```

Comment:
The text here really does need to change. The simplest change is from 'Unlike the' to 'Unlike other'. If you want to provide more detail, you could explain that all other cipher strings are "+" separated conjunctions (AND) of basic cipher strings, which represent a single cipher or class of ciphers sharing a common elementary property. The "DEFAULT" keywork is not such a cipher string, it is instead a built-in cipher list definition that can be a starting point for further refinement, but is not combinable with "+", and must come first.

Reply:
Re s/Unlike the/Unlike other/, the way things are expressed here, `DEFAULT` isn't a cipher string, but rather a prefix.  Seen that way, 'other' would be wrong, wouldn't it?  Although, I might want to see that changed to 'Unlike cipher strings, ...'